pages/glossary/semantic-link.md

```diff
@@ -5,7 +5,7 @@ unambiguous: true
 objective: true
 ---
 
-A semantic link is any HTML element which has the [semantic role][] of ['link'][link] or a [semantic role][] that inherits from the ['link'][link] role.
+A semantic link is any HTML element which has the [semantic role][] of [`link`][] or a [semantic role][] that inherits from the [`link`][] role.
```

Comment:
I think you need to keep these because [`link`] is not the same as [link]

```suggestion
A semantic link is any HTML element which has the [semantic role][] of [`link`][link] or a [semantic role][] that inherits from the [`link`][link] role.
```

Reply:
Pretty sure that would work, but I'm not bothered either way.